graphql_compiler/macros/macro_edge/helpers.py

```diff
@@ -123,22 +124,25 @@ def _replace_tag_names_in_filter_directive(name_change_map, filter_directive, as
         elif argument.name.value == 'value':
             new_value_list = []
             for value in argument.value.values:
-                if is_tagged_parameter(value.value):
-                    current_name = get_parameter_name(value.value)
+                parameter = value.value
+                if is_tagged_parameter(parameter):
+                    current_name = get_parameter_name(parameter)
                     new_name = name_change_map[current_name]
                     if new_name != current_name:
                         made_changes = True
                     new_value_list.append(StringValue('%' + new_name))
-                elif is_runtime_parameter(value.value):
+                elif is_runtime_parameter(parameter):
                     new_value_list.append(value)
                 else:
-                    raise AssertionError(u'Value was neither tag nor filter: {} {}'
-                                         .format(filter_directive, value.value))
+                    raise AssertionError(u'Parameter {} used in directive {} is expected to '
+                                         u'be a tagged parameter (starting with %) or a runtime '
+                                         u'parameter (starting with $), but it was neither. {}',
```

Comment:
You accidentally deleted the `.format()` here, and are instead passing 4 arguments to the `AssertionError` constructor.